cpp/test/torch_scripted/torch_scripted_test.cc

```diff
@@ -9,44 +9,47 @@
 
 TEST_F(ModelPredictTest, TestLoadPredictBaseHandler) {
   this->LoadPredict(std::make_shared<torchserve::LoadModelRequest>(
-                        "test/resources/examples/mnist/mnist_handler",
+                        "_build/test/resources/examples/mnist/mnist_handler",
                         "mnist_scripted_v2", -1, "", "", 1, false),
-                    "test/resources/examples/mnist/base_handler",
-                    "test/resources/examples/mnist/0_png.pt", "mnist_ts", 200);
+                    "_build/test/resources/examples/mnist/base_handler",
+                    "_build/test/resources/examples/mnist/0_png.pt", "mnist_ts",
+                    200);
 }
 
 TEST_F(ModelPredictTest, TestLoadPredictMnistHandler) {
   this->LoadPredict(std::make_shared<torchserve::LoadModelRequest>(
-                        "test/resources/examples/mnist/mnist_handler",
+                        "_build/test/resources/examples/mnist/mnist_handler",
                         "mnist_scripted_v2", -1, "", "", 1, false),
-                    "test/resources/examples/mnist/mnist_handler",
-                    "test/resources/examples/mnist/0_png.pt", "mnist_ts", 200);
+                    "_build/test/resources/examples/mnist/mnist_handler",
+                    "_build/test/resources/examples/mnist/0_png.pt", "mnist_ts",
+                    200);
 }
```

Comment:
What's the difference b/w these 2 test cases?
- base handler folder
- mnist handler folder

Reply:
mnist handler has a slightly different postprocessing (applies argmax). In a follow up PR we should also change the preprocessing step into accepting an image instead of a .pt file.